examples_tests/20.Megatexture/main.cpp

```diff
@@ -521,7 +522,7 @@ APP_CONSTRUCTOR(MeshLoadersApp)
             vt->commit(cm.addr, cm.texture.get(), cm.subresource, cm.uwrap, cm.vwrap, cm.border);
         }
 
-        auto gpuvt = core::make_smart_refctd_ptr<video::IGPUVirtualTexture>(engine->logicalDevice.get(), engine->queues, vt.get());
+        auto gpuvt = core::make_smart_refctd_ptr<video::IGPUVirtualTexture>(engine->logicalDevice.get(), engine->gpuTransferFence.get(), engine->queues[CommonAPI::InitOutput<1>::EQT_TRANSFER_UP], vt.get());
```

Comment:
fence and queue cannot go into the constructor @Przemog1

Reply:
ah ok its the constructor...

you should just take the queue, and make your own fence in there.